.github/workflows/testing/push.json

```diff
@@ -1,10 +1,10 @@
 {
     "push": {
       "head": {
-        "ref": "users/awentzel/update-action-cc"
+        "ref": "users/awentzel/test-prettier-failure"
       },
       "base": {
-        "ref": "users/awentzel/update-action-cc"
+        "ref": "users/awentzel/test-prettier-failure"
```

Comment:
just a note that there are methods for testing github actions locally and we should probably remove any temporary or testing files if possible.

Reply:
I'm using one method to test locally.  This file is required, but, will change per test, but is not autogenerated.  What do you suggest?

Reply:
A cursory look came up with https://github.com/nektos/act

I'm not sure myself what else is out there but if we're committing to the repository test branches to check CI/CD we might look into something like the link above instead of testing it through the repository on github. This is assuming that I'm understanding this correctly and this user branch is part of a test you're running to double check CI/CD. If it's not at this is something else, let me know, but that's how it reads.

Reply:
I think you are misunderstanding, @janechu.  I already use nektos/act and this file is based on how you configure it. Please see the details in our repository `.github/workflows/testing/README.md`

Reply:
That makes sense, this is just a suggestion to remove the local branch reference since it is only useful for someone who has this branch locally and is using it for testing.

Reply:
#3041 will address @janechu concerns.